app/views/users/invite.erb

```diff
@@ -17,27 +17,42 @@
     <%= f.text_field :family_name, class: "govuk-input" %>
   </div>
 
-  <div class="govuk-form-group">
-  <fieldset class="govuk-fieldset">
-    <div class="govuk-checkboxes">
-      <div class="govuk-checkboxes__item">
-        <%= f.check_box(:roles, { :class => 'govuk-checkboxes__input', :multiple => true }, ROLE::CERTIFICATE_MANAGER, nil) %>
-        <%= f.label :roles, t("users.roles.#{ROLE::CERTIFICATE_MANAGER}"), class: 'govuk-label govuk-checkboxes__label' %>
-      </div>
-      <div class="govuk-checkboxes__item">
-        <%= f.check_box(:roles, { :class => 'govuk-checkboxes__input', :multiple => true }, ROLE::USER_MANAGER, nil) %>
-        <%= f.label :roles, t("users.roles.#{ROLE::USER_MANAGER}"), class: 'govuk-label govuk-checkboxes__label' %>
-      </div>
-      <!-- TODO: hide or move away for non-gds users -->
-      <div class="govuk-checkboxes__item">
-        <%= f.check_box(:roles, { :class => 'govuk-checkboxes__input', :multiple => true }, ROLE::GDS, nil) %>
-        <%= f.label :roles, 'GDS', class: 'govuk-label govuk-checkboxes__label' %>
-      </div>
+  <% if @gds_team %>
+    <div class="govuk-form-group">
+      <fieldset class="govuk-fieldset">
+        <div class="govuk-checkboxes">
+          <div class="govuk-checkboxes__item">
+            <%= f.check_box(:roles, { :class => 'govuk-checkboxes__input', :multiple => true, :checked => true, :disabled => true }, ROLE::GDS, nil) %>
```

Comment:
Using old syntax? Should be `class: 'govuk-checkboxes__input'` etc instead?

Reply:
ah yeah, too much copy-paste 🤐